examples/prisma-react/src/TodoList.telefunc.ts

```diff
@@ -0,0 +1,44 @@
+import { Abort, shield } from 'telefunc'
+import prisma from './client'
+
+const t = shield.type
+
+export async function onGetTodos() {
+  const todos = await prisma.todo.findMany()
+  return todos
+}
+
+export const onToggleTodo = shield([t.number], async function onToggleTodo(id) {
+  const todo = await prisma.todo.findUnique({
+    where: {
+      id,
+    },
+  })
+  if (!todo) {
+    throw Abort('Todo not found')
```

Comment:
It should be `throw Abort()` instead. See https://telefunc.com/permissions and https://telefunc.com/abort-vs-error